src/client/components/Reviews.jsx

```diff
@@ -22,29 +22,30 @@ class Reviews extends React.Component {
     return reviews.slice(maxIndex - 20, maxIndex);
   }
 
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+    const { restaurantId } = props;
     this.state = {
       filteredReviews: null,
       reviewsToDisplay: null,
+      restaurantId,
     };
     this.filters = {
       sort: 'Newest',
       stars: null,
       categories: new Set(),
       page: 1,
     };
-    this.reviews = sampleRestaurant.reviews;
+    this.reviews = null;
     this.sortReviews = this.sortReviews.bind(this);
     this.filterStars = this.filterStars.bind(this);
     this.filterPage = this.filterPage.bind(this);
     this.filterCategories = this.filterCategories.bind(this);
   }
 
   componentDidMount() {
-    const { pathname } = window.location;
-    const restId = window.location.pathname.substring(13, pathname.length - 1);
-    fetch(`http://localhost:1337/restaurants/${restId}/reviews`)
+    const { restaurantId } = this.state;
```

Comment:
Is adding the ID to the state redundant when you can access directly from the props here instead?